src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -1202,43 +1157,49 @@ public List<String> getInactiveConsumerIds(Instant lastCheckedInRetention, Insta
      *     Org namespace to search
      * @return Consumer that matches the given
      */
-    @SuppressWarnings("unchecked")
     @Transactional
-    public CandlepinQuery<Consumer> getHypervisorsBulk(Iterable<String> hypervisorIds, String ownerId) {
+    public List<Consumer> getHypervisorsBulk(Collection<String> hypervisorIds, String ownerId) {
         if (hypervisorIds == null || !hypervisorIds.iterator().hasNext()) {
```

Comment:
With the type change, we can update this validation to use `.isEmpty` rather than pulling an iterator.

Reply:
done